### DISCUSSION TITLE
is there a way to edit URLs?

### DISCUSSION THREAD

**Discussion by TheScorpio32** [redimp/otterwiki#130] [Q&A] [answered]
Hello, I've been trying out establish a wiki for a game I am playing and so far Otterwiki has been great.
I was just wondering if there is a way to remove %20 from URLs? Unfortunately my page title has to have multiple words in it. so the url is showing like this.

example.com/Open%20World%20Title

**Comment by redimp** [comment, marked as answer]
Hey @TheScorpio32, unfortunately this is the way how spaces are encoded in URIs, see the [rfc3986](https://www.rfc-editor.org/rfc/rfc3986#section-2.1). Currently An Otter Wiki does not provide a  workaround for this in the URI available. 

When it comes to write the links, An Otter Wiki provides wikilinks, for example `[[Open World Title]]`, here you can use unencoded spaces.

Another Option would be to name pages in CamelCase, like `https://example.com/OpenWorldTitle`.
